components/navbar/index.ts

```diff
@@ -173,6 +174,11 @@ export class QiskitNavbar extends LitElement {
                 </li>
               `
             )}
+            <li href="/account" class="navbar__menu__account">
+              <a href="/account" class="navbar__menu__account__link">
```

Comment:
Extract the account link as the `_homeLink`.

Reply:
I created a new object called _useLink